packages/core/src/components/text-field/_text-field.scss

```diff
@@ -3,7 +3,9 @@
 
 @include exports('ray-text-field') {
   .#{$ray-class-prefix}text-field {
+    @include required-support('text-field');
```

Comment:
noticed Ray input labels didnt really have support for adding (*) to required fields

Reply:
Cool!